.github/workflows/build.yml

```diff
@@ -0,0 +1,79 @@
+name: Build
+
+on: [ push, pull_request, workflow_dispatch ]
+
+jobs:
+  node:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up chromedriver
+        uses: nanasess/setup-chromedriver@master
+
+      - run: |
+         sudo apt-get install xvfb
```

Comment:
Looks as if setting up chrome driver twice? `nanasess/setup-chromedriver@master`

Reply:
Removed usage of `nanasess/setup-chromedriver@master`